src/StaticFAQ/FAQCategoryList.js

```diff
@@ -2,66 +2,138 @@
 
 import * as React from 'react';
 import idx from 'idx';
+import { Link } from 'react-router-dom';
 import { graphql, QueryRenderer } from 'react-relay';
 
-import FAQCategory from './FAQCategory';
+import { FAQCategory, FAQArticle } from './FAQItem';
 import environment from '../relay/environment';
-import type { FAQCategoryListQueryResponse } from './__generated__/FAQCategoryListQuery.graphql';
+import routeDefinitions from './../routeDefinitions';
+import type { FAQCategoryListRootQueryResponse } from './__generated__/FAQCategoryListRootQuery.graphql';
+import type { FAQCategoryListSubcategoryQueryResponse } from './__generated__/FAQCategoryListSubcategoryQuery.graphql';
 
-type Props = {};
+type Props = {|
+  categoryId: string | null,
+|};
 
-type QueryRendererParams = {
-  props: FAQCategoryListQueryResponse,
+type RootQueryRendererParams = {
+  props: FAQCategoryListRootQueryResponse,
   error: Error,
 };
 
-const query = graphql`
-  query FAQCategoryListQuery {
+type SubcategoryQueryRendererParams = {
+  props: FAQCategoryListSubcategoryQueryResponse,
+  error: Error,
+};
+
+const queryRoot = graphql`
+  query FAQCategoryListRootQuery {
     allFAQCategories(language: en) {
       edges {
         node {
           id
-          ...FAQCategory_category
+          ...FAQItemCategory
         }
       }
     }
   }
 `;
+const querySubcategory = graphql`
+  query FAQCategoryListSubcategoryQuery($id: ID!) {
+    FAQCategory(id: $id, language: en) {
+      subcategories {
+        id
+        ...FAQItemCategory
+      }
+      FAQs {
+        id
+        ...FAQItemArticle
+      }
+    }
+  }
+`;
 
 class FAQCategoryList extends React.Component<Props> {
-  renderStaticFAQs = (rendererProps: QueryRendererParams) => {
+  renderFAQArticlePerexes = (faqs: Object[]) => {
+    return <div>{faqs.map(faq => <FAQArticle key={faq.id} data={faq} />)}</div>;
+  };
+
+  renderCategories = (categories: Object[]) => {
```

Comment:
We can probably 
```
import type { FAQItemCategory}  from './__generated__/FAQItemCategory.graphql';
```
And use that type instead of `Object` here.

Reply:
good point, fixed